Support/ceptr/ceptr/ceptr.py

```diff
@@ -1,45 +1,91 @@
 """Convert cantera mechanism to C++ files."""
 import argparse
 import pathlib
+from argparse import RawTextHelpFormatter
 
 import cantera as ct
 
 import ceptr.converter as converter
 
 
-def convert(fname):
+def convert(
+    fname,
+    format_input,
+    symbolic_jac,
+):
     """Convert a mechanism file."""
     mechanism = ct.Solution(fname)
-    conv = converter.Converter(mechanism)
+    conv = converter.Converter(
+        mechanism,
+        format_input,
+        symbolic_jac,
+    )
     conv.writer()
     conv.formatter()
 
 
-def convert_lst(lst):
+def convert_lst(
+    lst,
+    format_input,
+    symbolic_jac,
+):
     """Convert mechanisms from a file containing a list of directories."""
     lpath = pathlib.Path(lst)
     with open(lst, "r") as f:
         for line in f:
             if not line.startswith("#"):
                 mechname = lpath.parents[0] / line.strip()
                 print(f"""Converting file {mechname}""")
-                convert(mechname)
+                convert(
+                    mechname,
+                    format_input,
+                    symbolic_jac,
+                )
 
 
 def main():
     """Convert cantera mechanisms to C++ files."""
-    parser = argparse.ArgumentParser(description="Mechanism converter")
+    parser = argparse.ArgumentParser(
+        description="Mechanism converter", formatter_class=RawTextHelpFormatter
+    )
     group = parser.add_mutually_exclusive_group(required=True)
     group.add_argument("-f", "--fname", help="Mechanism file", type=str)
     group.add_argument(
         "-l", "--lst", help="Mechanism directory file list", type=str
     )
+
+    parser.add_argument(
+        "--format_input",
+        help=(
+            "Input file for jacobian formatting parameters used with QSS"
+            " mechanisms"
+        ),
+        type=str,
+        default=None,
+        required=False,
+    )
+
+    parser.add_argument(
+        "-sj",
+        "--symbolic_jacobian",
+        action="store_true",
+        help="Compute the Jacobian using symbolic recording",
```

Comment:
Is this only for QSS? Maybe add that to the help string? Maybe the option should be `qss_symbolic_jacobian`?